docs/application/dotnet/guides/app-management/widget-control.md

```diff
@@ -0,0 +1,130 @@
+# Widget Control
+
+The widget control provides information about installing and running widget applications.
+
+It also provides functions for the following:
+
+-   Sending update requests to the widget applications
+-   Retrieving details of running instance for the same package widget applications
+
+The main features of the `Tizen.Applications.WidgetControl` class include:
+
+-   [Getting information on widget applications](#getting_information)
+
+    For the widget applications that is installed but not running, you can retrieve widget information.
+
+-   [Listening widget lifecycle events on widget applications](#listening_events)
+
+    For running widget applications, you can listen widget application lifecycle events.
+
+-   [Communicating with running widget instances](#communicating_instances)
+
+    For running widget instances, you can trigger an update event, send update data, and retrieve running widget instances detail.
+
+## Prerequisites
+
+To enable your application to use the widget control functionality:
+
+1.  To use the methods and properties of the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class, include the [Tizen.Applications](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.html) namespace in your application:
+
+    ```
+    using Tizen.Applications;
+    ```
+
+2.  To get information on widget application, the application has to request permission by adding the following privilege to the  `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/widget.viewer</privilege>
+    </privileges>
+    ```
+
+<a name="create_instance"></a>
+## Creating instance of Widget Control
+
+To create an instance of the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class with the ID of the widget application, as shown in the following code:
+
+    ```
+    WidgetControl control = new WidgetControl(Your Widget ID);
+    ```
+
```

Comment:
Please add:
2. Operate on the control:

- Get the main application ID, package ID, and available scale lists from the control:
    ```
    string mainId = control.MainAppId;
    string packageId = control.PackageId;
    IEnumerable<WidgetControl.Scale> scales = control.GetScales();
    ```

Reply:
@mijin85cho, This comment which I had posted must be ignored. 
I forgot to delete this comment.